backend/app/models.py

```diff
@@ -36,3 +36,15 @@ class GerryDBTableBase(TimeStampMixin, SQLModel):
 class GerryDBTable(GerryDBTableBase, table=True):
     uuid: str = Field(sa_column=Column(UUIDType, unique=True))
     name: str = Field(nullable=False, unique=True)
+
+
+class Document(TimeStampMixin, SQLModel):
+    document_id: str | None = Field(sa_column=Column(UUIDType, unique=True))
+
+
+class Assignments(SQLModel, table=True):
+    # this is the empty parent table; not a partition itself
+    document_id: str = Field(foreign_key="document.document_id", primary_key=True)
+    geo_id: str = Field(primary_key=True)
```

Comment:
Also did we want to add indices to each of these?
```suggestion
    document_id: str = Field(foreign_key="document.document_id", primary_key=True, index=True)
    geo_id: str = Field(primary_key=True, index=True)
```
Or just on `geo_id` since the table is partitioned?

Reply:
just geoid; an index on document_id is redundant with the partition.